peps/pep-0776.rst

```diff
@@ -72,6 +72,37 @@ Packaging Goals
 Emscripten Platform Information
 ===============================
 
+"Pyodide" vs "Emscripten Python"
+--------------------------------
+
+For the sake of this document, we use the term "Emscripten Python" to refer to
+the Emscripten Python maintained in the ``python/cpython`` repository, without
+any downstream additions. We contrast the features present in Emscripten Python
+to the features present in Pyodide.
+
+Pyodide is maintained `on github <https://github.com/pyodide/pyodide>`__ and
+distributed via `jsDelivr <https://www.jsdelivr.com/oss-cdn/pyodide>`__, `npm
+<https://www.npmjs.com/package/pyodide>`__, and `github releases
+<https://github.com/pyodide/pyodide/releases>`__.
+
+Emscripten Python is not distributed, but it is possible to build and test by
+executing the following commands from the root of the CPython repository:
```

Comment:
Given this is a PEP, would it be preferable to point directly at the documentation in the CPython repo? The PEP is a standards document, and we're not standardising how Emscripten Python is built; encoding the instructions here could be interpreted as "...and this is how it will always be built".

Reply:
I suppose I'll make a PR to add an Emscripten section to the dev guide and then point at that.